addons/purchase_stock/views/product_views.xml

```diff
@@ -8,7 +8,7 @@
             <field name="property_account_income_categ_id" position="before">
                 <field name="property_account_creditor_price_difference_categ"
                     domain="[('deprecated','=',False)]" groups="account.group_account_readonly"
-                    invisible="property_valuation == 'manual_periodic'"/>
+                    invisible="property_valuation == 'manual_periodic' or property_cost_method != 'standard'"/>
```

Comment:
@thomasdeleval bad idea. Price diff could be use in that case https://www.odoo.com/documentation/17.0/applications/finance/accounting/get_started/avg_price_valuation.html?highlight=valuation#on-product-return
@rare-odoo could you remove this commit from the PR

Reply:
indeed